services/hh/config.go

```diff
@@ -23,11 +23,15 @@ const (
 	// attempting to flush hinted handoff queues. With each failure of a hinted
 	// handoff write, this retry interval increases exponentially until it reaches
 	// the maximum
-	DefaultRetryInterval = time.Second
+	DefaultRetryInterval = 10 * time.Second
```

Comment:
Would be great if this was lower (like 1s or 100ms) and the backoff would get it up this level quick